generators/typegen/generator.go

```diff
@@ -126,15 +158,57 @@ func (g *Generator) generateTypesForPackage(s *schema.Schema, genConfig *config.
 			}
 
 			enumsForGen = append(enumsForGen, xxx)
+		case schema.KindScalar:
+			log.Warnf("scalar type: %+v", t)
+
+			// Default scalars to int
+			createAs := "int"
```

Comment:
Shouldn't these default to `string`?

Reply:
I don't know, should they be?

Reply:
Updated to string.  I don't understand scalars well enough to know if this is right.  You'd think that if the schema wanted a string, they would use a string instead of a scalar.

Reply:
My thought here is that you can put more "stuff" into a string than an int, including all the default [scalar types](https://graphql.org/learn/schema/#scalar-types) that GraphQL implements.

Reply:
Makes sense to me.